tests/unit/dataframer/test_dataframer_smmart.py

```diff
@@ -1,133 +1,257 @@
     return {'identifier': '9ae7e542-767f-4b03-a854-7ceed17152cb',
             'resourceType': 'DocumentReference',
             'id': '9ae7e542-767f-4b03-a854-7ceed17152cb',
-            'status': 'current', 'docStatus': 'final',
+            'status': 'current',
+            'docStatus': 'final',
             'date': '2024-08-21T10:53:00+00:00',
             'md5': '227f0a5379362d42eaa1814cfc0101b8',
-            'title': 'specimen_1234_labA.fq.gz',
             'source_path': 'file:///home/LabA/specimen_1234_labA.fq.gz',
             'contentType': 'text/fastq',
-            'size': 5595609484, 'title': 'specimen_1234_labA.fq.gz',
+            'url': 'file:///home/LabA/specimen_1234_labA.fq.gz',
+            'size': 5595609484,
+            'title': 'specimen_1234_labA.fq.gz',
             'creation': '2024-08-21T10:53:00+00:00',
             'sequencer': 'Illumina Seq 1000',
             'index': '100bp Single index', 'type': 'Exome',
             'project_id': 'labA_projectXYZ',
             'read_length': '100',
             'instrument_run_id': '234_ABC_1_8899',
             'capture_bait_set': 'Human Exom 2X',
-            'end_type': 'Paired-End', 'capture': 'emitter XT',
+            'end_type': 'Paired-End',
+            'capture': 'emitter XT',
             'sequencing_site': 'AdvancedGeneExom',
             'construction': 'library_construction',
-            'Gene': 'TP53', 'Chromosome': 'chr17',
+            'Gene': 'TP53',
+            'Chromosome': 'chr17',
             'result': 'gain of function (GOF)',
             'specimen_collection': 'Breast',
             'specimen_id': '60c67a06-ea2d-4d24-9249-418dc77a16a9',
             'specimen_identifier': 'specimen_1234_labA',
-            'specimen_processing': 'Double-Spun',
-            'url': 'file:///home/LabA/specimen_1234_labA.fq.gz'
+            'specimen_processing': 'Double-Spun'
             }
 
+@pytest.fixture()
+def smmart_observation_dataframes():
+    """Create user-defined dataframe/row with Observations (and user-defined nomenclature) that have focus/association to this DocumentReference instance"""
+    return [{
+        "identifier": "patientX_1234-9ae7e542-767f-4b03-a854-7ceed17152cb-sequencer",
+        "resourceType": "Observation",
+        "id": "cec32723-9ede-5f24-ba63-63cb8c6a02cf",
+        "status": "final",
+        "category": "Laboratory",
+        "sequencer": "Illumina Seq 1000",
+        "index": "100bp Single index",
+        "type": "Exome",
+        "project_id": "labA_projectXYZ",
+        "read_length": "100",
+        "instrument_run_id": "234_ABC_1_8899",
+        "capture_bait_set": "Human Exom 2X",
+        "end_type": "Paired-End",
+        "capture": "emitter XT",
+        "sequencing_site": "AdvancedGeneExom",
+        "construction": "library_construction",
+        "documentreference_identifier": "9ae7e542-767f-4b03-a854-7ceed17152cb",
```

Comment:
Wouldn't we be creating a dataframe on a document reference and include observations focusing on it?

Reply:
Yes, see `test_flatten_document_references`

Reply:
@quinnwai  can't find it
```
$ git log --oneline | head -1
4001bd9 fix warning on test, rename test dataframe file
$ grep -r test_flatten_document_references tests || echo 'cant find it'
cant find it
```

Reply:
Typo, it was test_flatten**ed**_document_references, see [here](https://github.com/ACED-IDP/gen3_util/blob/4001bd95c5da21d71520c6b0e758c6109afa7c56/tests/unit/dataframer/test_dataframer.py/#L362-L372).